docs/whats_new.rst

```diff
@@ -19,17 +19,23 @@ People who contributed to this software across releases (in alphabetical order)
 
 * `Aamna Lawrence`_
 * `Adam Li`_
+* `Christian Oreilly`_
 * `Stefan Appelhoff`_
 * `Victor Xiang`_
 * `Yorguin Mantilla`_
 
+
 .. _current:
 
 Current
 -------
 
 Changelog
 ~~~~~~~~~
+- Adding ``chs_types`` and ``chs_non_eeg`` attributes to ``PrepPipeline``, by `Yorguin Mantilla`_ (`#34 <https://github.com/sappelhoff/pyprep/pull/34>`_)
```

Comment:
as also mentioned in a comment below: We should properly document all publicly accessible class attributes in the class docstr in an `Attributes` section.

Reply:
Should those attributes be public though, I mean should anyone actually use the channel names and types attributes?